python/test/unit/language/test_core.py

```diff
@@ -645,6 +645,10 @@ def expand_dims_kernel(dummy, N: tl.constexpr):
         t = tl.expand_dims(scalar, -1)
         tl.static_assert(t.shape == [1])
 
+        # N is a scalar that's not even a tl.tensor -- this should work too.
+        t = tl.expand_dims(N, -1)
```

Comment:
It could be handled already using `tl.full` right?

Reply:
Yes, but it's confusing because most other Triton ops already work when passed "real" scalars.

Reply:
Got it